net/unbound/Makefile

```diff
@@ -37,9 +37,9 @@ define Package/unbound
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=IP Addresses and Names
+  USERID:=unbound=553:unbound=553
```

Comment:
From my day job there is such a thing as testing delta from a *stable* platform. I realize its a conservative philosophy, but its a professional one, and if it isn't fixed in an official release, then it isn't fixed. I hate risking a whole trunk build just to test a stand-alone plug-in package. 

15.05.1 is the last release, and its broke. I am talking completely new install of CC. Boot. Opkg install unbound.ipk ('n friends). UCI dnsmasq out of the way. Start unbound. Unbound crashes because it doesn't have a user. /lib/functions.sh has a defect where it doesn't pick a user number, but creates some mess in /etc/passwd. Hand modify /etc/passwd and we are running.

*(side note, /lib/functions.sh is rather a mess. It should at least be two functions okpg-helper, uci-helper, and then whatever else is stuffed in there.)*